pkg/server/server_test.go

```diff
@@ -1,15 +1,23 @@
 package server
 
 import (
+	"context"
 	"io/ioutil"
 	"net/url"
 	"os"
 	"testing"
 
+	"fmt"
```

Comment:
"fmt" and "bytes" should be grouped with the rest of the `std` imports

Reply:
done